xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
According to http://gcc.gnu.org/onlinedocs/gcc-2.95.3/cpp_1.html#SEC6 , the only actual difference is that `""` also searches the current dir.

So yeah, `<>`is the proper way for any include not in the build tree, really.